packages/react-day-picker/src/hooks/useDayModifiers/useDayModifiers.ts

```diff
@@ -0,0 +1,51 @@
+import { ModifierStatus } from '../../types/Modifiers';
+import { useDayPicker } from '../../contexts/DayPicker';
+import { useModifiers } from '../../contexts/Modifiers';
+import { getModifierStatus } from './utils/getModifierStatus';
+
+export type DayModifiers = {
+  /** The status of the modifiers */
+  modifiers: ModifierStatus;
+  /** The class names based on modifiers. */
+  modifierClassNames: string[];
+  /** The style based on modifiers. */
+  modifierStyle: React.CSSProperties;
+};
+
+/** Return the modifiers and its styles for the specified date. */
+export function useDayModifiers(date: Date): DayModifiers {
```

Comment:
```suggestion
/** Return modifiers and styles for the specified day. */
export function useDayModifiers(day: Date): DayModifiers {
```

Reply:
The line between what a "date" and what a "day" is is not clear to me. In Day.tsx, we seem to take a `date` and convert it into a `Day`:
```
const day = useDay(props.date, props.displayMonth, buttonRef);
```

useDay also uses `date`:
```
export function useDay(
  /** The day rendered in the month. */
  date: Date,


```
  const { modifiers, modifierClassNames, modifierStyle } =
    useDayModifiers(date);
```

So I think the parameter name should still be `date`

Reply:
ahah yeah its not clear at all. I think if a hook is called "useDay" than it should have an argument called "day". Otherwise – which day is "using"?